src/frameworks/web/components/organisms/Landing/LandingBanner.tsx

```diff
@@ -8,6 +8,8 @@ import {
   Row, Col, Visible, Hidden, ScreenClassRender,
 } from 'react-grid-system';
 
+import { ILandingContainer } from '../../../../../interfaces/frameworks/web/components/organisms/LandingBanner/ILandingBanner';
```

Comment:
절대경로로 되어있는데, 나중에 혹시나 문제가 있을 수 있으니, 아래처럼 바꾸는건 어떨까 싶습니당

```suggestion
import { ILandingContainer } from 'interfaces/frameworks/web/components/organisms/LandingBanner/ILandingBanner';
```